db/migrate/20190620092211_backfill_add_limit_exceeded_error_to_proxies.rb

```diff
@@ -0,0 +1,17 @@
+class BackfillAddLimitExceededErrorToProxies < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def up
+    Proxy.select(:id).find_in_batches(batch_size: 100) do |batch|
+      batch.each { |proxy| proxy.update_column(:error_headers_limits_exceeded, 'text/plain; charset=us-ascii') }
+      sleep(1)
```

Comment:
In SaaS we have around 33k proxies so it will last ~6 minutes
Can we lower that to 0.5 ?